addons/website_sale/views/templates.xml

```diff
@@ -1944,24 +1944,19 @@
                                         <t t-if="website._display_partner_b2b_fields()">
                                             <div class="w-100"/>
                                             <t t-set='vat_warning' t-value="'vat' in checkout and checkout['vat'] and not can_edit_vat" />
-                                            <t t-if="(mode == ('new', 'billing') and is_public_order
-                                                or mode == ('edit', 'billing') and partner_id == website_sale_order.partner_id.id)
-                                                and (can_edit_vat or 'vat' in checkout and checkout['vat'])"
-                                            >
```

Comment:
IIUC, you're removing this `t-if` without any other consideration?
Meaning you see the vat & company name always, regardless of the address type (meaningless for delivery addresses), the company configuration, ...

There is probably something that has to be updated in the `t-if` to consider cases where the vat has to be set and isn't, but it shouldn't be removed like that.

Reply:
Hello @Feyensv, I changed the fix here to add a variable to force show the vat with latam countries which has the mimimal xml change